_posts/2025-04-29-homebrew-4.5.0.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 4.5.0
+author: MikeMcQuaid
+redirect_from: /blog/4.5.0/
+---
+
+Today, I'd like to announce Homebrew 4.5.0.
+The most significant changes since 4.4.0 are major improvements to `brew bundle`/`services`, Linux support for Casks, official Support Tiers, the Tier 2 ARM64 Linux support, Ruby 3.4 and several other deprecations.
```

Comment:
```suggestion
The most significant changes since 4.4.0 are major improvements to `brew bundle`/`services`, Linux support for casks, official Support Tiers, Tier 2 ARM64 Linux support, Ruby 3.4 and several other deprecations.
```